itests/eth_block_hash_test.go

```diff
@@ -49,9 +49,6 @@ func TestEthBlockHashesCorrect_MultiBlockTipset(t *testing.T) {
 	// let the chain run a little bit longer to minimise the chance of reorgs
 	n2.WaitTillChain(ctx, kit.HeightAtLeast(head.Height()+50))
 
-	head, err = n2.ChainHead(context.Background())
-	require.NoError(t, err)
```

Comment:
Any particular reason to remove this?

Reply:
It defeats the point of waiting 50 blocks for the chain to stabilize.

Reply:
Not sure I follow the thinking? This is just a sanity check. Might be worthless but what point does it defeat exactly?

Reply:
Unless we wait a bit after getting the head, there's no guarantee that `EthGetBlockByNumber` and `EthGetBlockByHash` will return the same value at the same height due to reorgs.

I _thought_ that that was the entire point of waiting an extra 50 blocks. If that wasn't, the comment:

> let the chain run a little bit longer to minimise the chance of reorgs

Is wrong.

Reply:
I see, all cleared up. `head` is being used below to traverse the chain up until that point. We do not need to traverse the extra 50 tipsets, those are there for stability. This is fine.